.circleci/config.yml

```diff
@@ -995,6 +1001,7 @@ workflows:
       - pynamodb: *requires_pre_test
       - pyodbc: *requires_pre_test
       - pyramid: *requires_pre_test
+      - pytest: *requires_pre_test
```

Comment:
Can we add this also under the `requires_tests` entry above?

Reply:
sure. can you please add?